.coderabbit.yaml

```
@@ -0,0 +1,130 @@
+    scope: "auto"
+  issues:
+    scope: "auto"
+  jira:
+    usage: "disabled"
+  linear:
+    usage: "disabled"
+  pull_requests:
+    scope: "auto"
+code_generation:
+  docstrings:
+    language: "en-US"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)